updates/99-fas.update

```diff
@@ -91,3 +91,18 @@ default:objectClass: groupOfPrincipals
 default:objectClass: ipaKrb5DelegationACL
 default:cn: fasjson-http-delegation
 default:ipaAllowedTarget: cn=ipa-ldap-delegation-targets,cn=s4u2proxy,cn=etc,$SUFFIX
+
+# create the container for the user agreements
+dn: cn=agreements,$SUFFIX
+default: objectclass: top
+default: objectclass: nsContainer
+default: cn: agreements
+plugin: update_managed_permissions
```

Comment:
The plugin line must come last (perhaps add a comment so other people don't add stuff after the plugin line?)